src/main/java/io/opentracing/contrib/spanmanager/tracer/ManagedSpanBuilder.java

```diff
@@ -101,8 +100,14 @@ public SpanBuilder withStartTimestamp(long microseconds) {
         return rewrap(delegate.withStartTimestamp(microseconds));
     }
 
+    /**
+     * @return Empty set of baggage items.
+     * @deprecated Don't call this anymore, SpanBuilder and SpanContext are separated since opentracing-api v0.22.0.
+     */
     public Iterable<Map.Entry<String, String>> baggageItems() {
```

Comment:
I thought about that, but then people will have to upgrade all jars at once.
Now the call will print warnings but won't break existing code (although admittedly bad code in the first place 😉).
I can remove it if you prefer?